dumper/app/build.gradle

```diff
@@ -41,6 +41,7 @@ dependencies {
 
     annotationProcessor libs.auto.value
 
+    implementation("org.apache.avro:avro:1.11.4")
```

Comment:
You already defined the version in [libs.versions.toml](https://github.com/google/dwh-migration-tools/pull/666/files#diff-697f70cdd88ba88fe77eebda60c7e143f6ad1286bca75017421e93ad84fb87df), so you can use reference here.

Reply:
will delete this line. duplication. line 54 does the same thing